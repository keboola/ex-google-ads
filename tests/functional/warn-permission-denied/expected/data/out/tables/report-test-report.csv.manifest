{
    "primary_key": [],
    "columns": [
        "campaignId",
        "campaignName",
        "metricsClicks",
        "metricsImpressions"
    ],
    "incremental": true
}