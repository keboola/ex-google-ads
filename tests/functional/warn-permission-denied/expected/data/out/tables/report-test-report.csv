"1015595176","test","0","0"
"6621804912","Brand awareness and reach-Display-1","0","0"
"6621889575","Brand awareness and reach-Display-1","0","0"
"11319146328","5f8014e5c9d74","0","0"
"11319151818","5f8015b83c676","0","0"
"11319214719","5f8018c6706da","0","0"
"11321411750","5f8014ed02dca","0","0"
"11321412164","5f8014e963314","0","0"
"11321415296","5f8015b4bc9eb","0","0"
"11321422487","5f8016aa4f085","0","0"
"11321424650","5f8016aeaa1b5","0","0"
"11321444618","5f8018c48f5d4","0","0"
"11321445245","5f8018c2cc944","0","0"
"11321787115","5f8015bceb339","0","0"
"11321795995","5f8016ac31a8d","0","0"
"11321827801","5f80193bab515","0","0"
"11321827906","5f80193971f77","0","0"
"11321828815","5f80193853166","0","0"
"12443401142","603ca2ff31563","0","0"
"12443401151","603ca3005ed55","0","0"
"12445768027","603c9d82b6090","0","0"
"12445812154","603ca3020a5f6","0","0"
"12445817197","603ca38abd8ca","0","0"
"12445817644","603ca38d330a0","0","0"
"12446272962","603c9d83d0272","0","0"
"12446273013","603c9d8544d4e","0","0"
"12446320737","603ca38bc8ff9","0","0"
"12449511071","603dfb13dbf92","0","0"
"12449950310","603e04a6c7426","0","0"
"12451943419","603dfb1531bd4","0","0"
"12451954233","603dfb170d3d5","0","0"
"12452363713","603e043da4b35","0","0"
"12452364901","603e043edc91e","0","0"
"12452373030","603e04402f67e","0","0"
"12452380783","603e04a4dfe62","0","0"
"12452380960","603e04a8df275","0","0"
"12913628868","608bf12352421","0","0"
"12913643754","608bf227cfda4","0","0"
"12913669434","608bf542028a6","0","0"
"12913671834","608bf5bca3829","0","0"
"12913683741","608bf713abded","0","0"
"12913685718","608bf7507e3d5","0","0"
"12913689249","608bf78aacb51","0","0"
"12913695000","608bf7f97e7e5","0","0"
"12913697001","608bf83bdfcb3","0","0"
"12913719087","608bfb21d9298","0","0"
"12913757686","608bf2cd899cf","0","0"
"12913782187","608bf562a704c","0","0"
"12913784074","608bf5a33065e","0","0"
"12913793008","608bf6ecb190b","0","0"
"12913794409","608bf733a1537","0","0"
"12913804993","608bf82f74a74","0","0"
"12913824880","608bfb1e9ba8b","0","0"
"12913824934","608bfb2fd9e02","0","0"
"12913828744","608bfba112272","0","0"
"12913828957","608bfba596b36","0","0"
"12914272034","608bef9dc1baa","0","0"
"12914294150","608bf1ea2d0fe","0","0"
"12914294849","608bf20670944","0","0"
"12914339240","608bf76abae12","0","0"
"12914342846","608bf7a3f2904","0","0"
"12914345270","608bf7d9d71ce","0","0"
"12914347619","608bf8149d7f6","0","0"
"12914349305","608bf85a63f96","0","0"
"12914371901","608bfba011b36","0","0"
"12945644865","608f928654655","0","0"
"12947238932","608f92853d2e0","0","0"
"12947340388","608f919b9fdf9","0","0"
"12947341108","608f91a178a59","0","0"
"12947341126","608f919d1bd67","0","0"
"12947344513","608f928ae446f","0","0"
"15102372693","617bc78719ced","0","0"
"15102880755","617be12f82695","0","0"
"15102881682","617be12b4e3e7","0","0"
"15103106304","617bed0f58666","0","0"
"15103106538","617bed0b1982d","0","0"
"15104003731","617bc7838ca94","0","0"
"15104426314","617bddfd704f5","0","0"
"15104426617","617bde01d01d3","0","0"
"15104445763","617bdf4938430","0","0"
"15104445820","617bdf4f4c250","0","0"
"15104459236","617be03e711f8","0","0"
"15104459440","617be04006c27","0","0"
"15104459740","617be0444a479","0","0"
"15104470768","617be12a0fb4d","0","0"
"15104669449","617bec6c43c6f","0","0"
"15104670214","617bec71540b8","0","0"
"15104678800","617bed0945331","0","0"
"15106182701","617bc7906e516","0","0"
"15106655207","617bddfbd7fcb","0","0"
"15106671782","617bdf4ab05b5","0","0"
"15106946360","617bec6a847ce","0","0"
"15233997531","618d789217472","0","0"
"15233999220","618d788fcb708","0","0"
"15234041700","618d7a37543bb","0","0"
"15234059436","618d7b2268372","0","0"
"15234093576","618d7c19b7aa2","0","0"
"15234094023","618d7c27abed9","0","0"
"15234095016","618d7c481fb6e","0","0"
"15234101964","618d7cdb58f4f","0","0"
"15234102705","618d7cf5ae934","0","0"
"15234116778","618d7d93ae0bb","0","0"
"15234146280","618d7eae27a5e","0","0"
"15234147252","618d7eb131113","0","0"
"15234148470","618d7ee455e2d","0","0"
"15234228924","618d813ce0f62","0","0"
"15234252135","618d827250276","0","0"
"15234254634","618d82bceb169","0","0"
"15234255111","618d82d109d07","0","0"
"15234258192","618d830942582","0","0"
"15234258603","618d831adbec8","0","0"
"15234268983","618d835263c5e","0","0"
"15234276459","618d83bd65d24","0","0"
"15234282930","618d84837a69b","0","0"
"15234283632","618d8485f16df","0","0"
"15237528451","618d7899b1e6e","0","0"
"15237539923","618d7901c54a2","0","0"
"15237540475","618d790b46f00","0","0"
"15237541816","618d79349c1e1","0","0"
"15237563491","618d7a34dfe74","0","0"
"15237578575","618d7b1fcc00a","0","0"
"15237597028","618d7c1031e04","0","0"
"15237598078","618d7c2aa4897","0","0"
"15237608878","618d7cd6ac68f","0","0"
"15237610459","618d7d118ff5d","0","0"
"15237610552","618d7d144c263","0","0"
"15237611164","618d7d2cca9d3","0","0"
"15237651817","618d7e9d3ba46","0","0"
"15237660205","618d7ee757c10","0","0"
"15237669070","618d7f5d8ae22","0","0"
"15237669274","618d7f6092bb3","0","0"
"15237722761","618d8288832b2","0","0"
"15237729484","618d828be0e94","0","0"
"15237736441","618d82a89bdc7","0","0"
"15237736450","618d82ab708de","0","0"
"15237743752","618d82d42ecb2","0","0"
"15237747061","618d830659dd4","0","0"
"15237757090","618d8329edcf5","0","0"
"15237757096","618d832d5e4bd","0","0"
"15237757864","618d8371a7651","0","0"
"15237758617","618d83429c0fb","0","0"
"15237774184","618d83c06cbc2","0","0"
"15237779914","618d848ba5ddb","0","0"
"15238900281","618e48fabcfff","0","0"
"15239664179","618d77b2dcb0b","0","0"
"15239693426","618d77aac05a0","0","0"
"15239693516","618d77a7e6a25","0","0"
"15239732351","618d790425e5a","0","0"
"15239733095","618d793830215","0","0"
"15239735504","618d793f98840","0","0"
"15239764046","618d7b05f0e34","0","0"
"15239764274","618d7b01aec7a","0","0"
"15239764763","618d7b2890a53","0","0"
"15239772128","618d7c12bc6e2","0","0"
"15239772917","618d7c455d5e0","0","0"
"15239773619","618d7c4f824d2","0","0"
"15239788274","618d7cf98a0a7","0","0"
"15239789648","618d7d31335a2","0","0"
"15239802122","618d7d97d49ec","0","0"
"15239812235","618d7e9a4f054","0","0"
"15239841323","618d7eeea586d","0","0"
"15239905382","618d813f7de43","0","0"
"15239906381","618d8145da5ea","0","0"
"15239947586","618d81e16a8bc","0","0"
"15239966351","618d82e5267fc","0","0"
"15239973284","618d82b9ca00a","0","0"
"15239974493","618d831d9d437","0","0"
"15239975027","618d82e7ed552","0","0"
"15239981510","618d833feecb6","0","0"
"15239981918","618d834fcf3fb","0","0"
"15239988650","618d83743f5d9","0","0"
"15243109852","618e4807a51ff","0","0"
"15243110596","618e480dd921f","0","0"
"15243152140","618e49b68ce4a","0","0"
"15245032004","618e48091eb78","0","0"
"15245063891","618e48f44f9e4","0","0"
"15245063948","618e48f6033aa","0","0"
"15245085995","618e49b0063b2","0","0"
"15245086076","618e49b1c5dd3","0","0"
